app/models/user.rb

```diff
@@ -240,7 +240,7 @@ def followed_articles
   end
 
   def cached_following_users_ids
-    Rails.cache.fetch(
+    RedisRailsCache.fetch(
       "user-#{id}-#{updated_at}-#{following_users_count}/following_users_ids",
       expires_in: 120.hours,
```

Comment:
While you are here, do you mind removing the extra comma?

Reply:
@mstruve that's going to be addede back in by Rubocop when the user commits the code, it's part of the configuration we have, see `Style/TrailingCommaInArguments`:

https://github.com/thepracticaldev/dev.to/blob/5135804a601b1ffa101289ae3841f118f0c4adbf/.rubocop.yml#L146-L150

The trailing comma will also mean one less diff line in case a new parameter is added :D

Reply:
TIL! Thanks!